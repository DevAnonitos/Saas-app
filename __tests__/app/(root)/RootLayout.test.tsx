import RootLayout from "@/app/(root)/layout";

describe('Page Imports', () => {
    it('should import RootLayout', () => {
        expect(RootLayout).toBeDefined();
        expect(RootLayout).toMatchSnapshot();
    });
});
