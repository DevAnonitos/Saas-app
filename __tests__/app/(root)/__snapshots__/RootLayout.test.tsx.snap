// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page Imports should import RootLayout 1`] = `[Function]`;
